UMA10 - Monitored Wallet Usage (#330)
NethermindEth/Forta-Agents: UMA-Bots/Wallet-Monitoring/README.md

# Monitored Wallet Usage detection bot

## Description

> This bot monitors for specific wallets (from a configurable list) that may use the [Across v2 bridge](https://across.to/) - a multichain bridge built on the [UMA protocol](https://umaproject.org/) as its source of on-chain data validation. For more details refer [here](https://discourse.umaproject.org/t/forta-monitors-across-v2-request-for-proposals/1569).

Reviewer: ```suggestion
This bot monitors specific wallets (from a configurable list) that may use the [Across v2 bridge](https://across.to/) - a multichain bridge built on the [UMA protocol](https://umaproject.org/) as its source of on-chain data validation. For more details refer [here](https://discourse.umaproject.org/t/forta-monitors-across-v2-request-for-proposals/1569).
```

## Supported Chains
- Mainnet
- Optimism
- Arbitrum
- Polygon
  
## Alerts

- UMA-10

>   - Fired whenever a monitored wallet uses the bridge

Reviewer: ```suggestion
  - Fired whenever a `FilledRelay` event is emitted from the `SpokePool` with a monitored wallet address as the depositor
```

  - Severity is always set to "low" 
  - Type is always set to "info"
  - Metadata :
      - `amount`: amount that the depositor wished to relay
      - `originChainId` : relay origination chain ID

>       - `destinationChainId` : LP Fee percentage computed by the relayer based on the deposit's quote timestamp and the HubPool's utilization.

Reviewer: ```suggestion
      - `destinationChainId` : relay destination chain ID
```
